src/network/taraxa_capability.cpp

```diff
@@ -1090,15 +1092,8 @@ void TaraxaCapability::logPacketsStats() {
 }
 
 void TaraxaCapability::onNewPbftVote(taraxa::Vote const &vote) {
-  std::vector<NodeID> peers_to_send;
-  {
-    boost::shared_lock<boost::shared_mutex> lock(peers_mutex_);
-    for (auto const &peer : peers_) {
-      if (!peer.second->isVoteKnown(vote.getHash())) {
```

Comment:
I guess you know what you're doing discarding this condition

Reply:
My mistake